### ISSUE TITLE
Insufficient Text/Background color contrast

### ISSUE THREAD

**Issue by jdeisenberg** [mprat/Terminus#95]
The site fails to pass WCAG guidelines for color contrast. This makes it difficult to use for anyone without extremely good vision.

**Comment by mprat** [comment]
Thanks for bringing this to my attention! I've never actually heard of these guidelines before - do you have a favorite resource for how to bring websites up to spec? Or a test procedure of some kind?

**Comment by jdeisenberg** [comment]
- Here’s a good article: https://webaim.org/articles/contrast/
- A specific page about color contrast with links to testing resources: https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum#resources
- I will ask around about resources for bringing websites up to spec and testing. Here is a list: https://www.w3.org/WAI/ER/tools/

**Comment by mprat** [comment]
Thank you very much! I'll definitely take a look at this - I've been meaning to give this a code reboot since it's quite ugly at the moment =)

**Comment by jdeisenberg** [comment]
I have gotten recommendations for:

- https://tenon.io/ (appears to be a commercial entity)
- Axe browser extension from https://www.deque.com/
- Bureau of Internet Accessibility (vendor) https://www.boia.org/about
- Wave at https://wave.webaim.org/
